sdk/core/azure-core/inc/http/http.hpp

```diff
@@ -3,27 +3,89 @@
 
 #pragma once
 
+#include <string>
+#include <vector>
+
 #include <internal/contract.hpp>
 
+using namespace std;
+
 namespace azure
 {
 namespace core
 {
 namespace http
 {
```

Comment:
The Storage SDK should be align with the format of this namespace. Can we confirm that the name space will be ALWAYS written in this way or in a single line like:
```
namespace azure {    namespace core {    namespace http {
```

Reply:
I'm not against it, but clangformat should allow that.

Reply:
@gilbertw

Reply:
My PR has a bunch of clang formating updates.  I'm not opposed to the single line namespaces if there is a way to do it in clang.  Another option is would be to put the { on the same line as the namespace.  This I know I can do in clang:
namespace azure {
namespace core {
namespace http {
  class HttpTransport
  {};

One other option clang supports easily is indention, though I don't like giving up all the whitespace on the left.
namespace azure {
  namespace core {
    namespace http {
      class HttpTransport
      {};

I vote for at least option 2, I'll get this on the list for design guidelines discussion this week

Reply:
Note I just figured out a way to do this with clang.  So here is another option

```
namespace azure { namespace core { namespace http {
  // TODO
  class HttpTransport
  {};
```

Reply:
> Note I just figured out a way to do this with clang. So here is another option

I like having at least one level of indentation for the fact that the class is within some set of namespace(s). + 1 on your latest suggestion :)

Reply:
We should never add anything to global namespace, so it is likely all our code will be indented.
It is not untypical for C++ to pass on namespaces spacing (https://www.boost.org/doc/libs/1_71_0/boost/smart_ptr/shared_ptr.hpp).

If we are adding just one indent for any number of namespaces, we are not gaining much - things on a same indentation level could be on different hierarchical levels across namespaces. But for sure, I do feel better about just one indent than multiple. I just don't think to always lose even one indent, in all our header files.